index.js

```diff
@@ -17,11 +19,19 @@ getFirstNames(usersArray);
 // Iteration 2 - `for...of` loop and ES6 string literals `${}`
 // ***************************************************************************
 
-const getFullNames = arr => {
+const getusers = arr => {
   // Your code goes here ...
+  const userusers = [];
+  for (let user of arr) {
+    let firstName = user.firstName;
+    let lastName = user.lastName;
```

Comment:
Try to use destructuring in cases like this.
const {firstName, lastName} = user